problems/1.types+operators.js

```diff
@@ -2,15 +2,20 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
+function isNumber(any) {
+  return typeOf (any) && "val"
+
+  
 
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+function isTruthy(any) {
+  let(any === 0);
+  return any
```

Comment:
You have to check for truthyness. Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.